scripts/db_migrator.py

```diff
@@ -632,8 +632,26 @@ def version_1_0_6(self):
             self.configDB.set_entry('DEVICE_METADATA', 'localhost', metadata)
             log.log_notice('Setting buffer_model to traditional')
 
-            self.set_version('version_3_0_0')
+            self.set_version('version_2_0_0')
 
+        return 'version_2_0_0'
+
+    def version_2_0_0(self):
+        """
+        Version 2_0_0
+        """
+        log.log_info('Handling version_2_0_0')
+        self.migrate_port_qos_map_global()
+        self.set_version('version_2_0_1')
+        return 'version_2_0_1'
+
+    def version_2_0_1(self):
```

Comment:
Here version 2_0_1 is making jump to version 3_0_0. However in 202012 branch version 2_0_1 makes jump to 2_0_2.

May be we should bring in changes made in https://github.com/sonic-net/sonic-utilities/pull/2277/files to master? That will keep different branches in sync.

Reply:
Unrelated to your changes, but general comment:
Inserting versions in between two existing versions also has another challenge - it causes two images (one from past and one latest) to have same DB versions, but different DBs.
For example, some images built with DB version 3_0_0 will have none of 202012 migration, and some will have all of the 202012 migration logic. This causes inconsistencies.

Reply:
Concluded in offline discussion. Solution is to port changes from 2277 to master after this PR merges.